src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -27,17 +29,19 @@ With just a few additions your existing Dockerfile can be used with our Node.js
    "newrelic": "latest",
    ```
 
-   Install a specific version, or use any of the other options provided by the [`package.json` format](https://docs.npmjs.com/files/package.json#dependencies). Check [the Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes) for information about past agent versions.
+You can update the value `latest` to install a specific version, or use any of the other options provided by the [`package.json` format](https://docs.npmjs.com/files/package.json#dependencies). Check [the Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes) for information about past agent versions.
```

Comment:
In the deploy preview, this section of text isn't flush as a child under the list. I'd shift tab both the code snippet and this piece of copy so it's nested underneath step 1.

Reply:
Ah, I wasn't sure how that got rendered. Let me try with my local copy...

Reply:
Finally fixed it! OK, hang on for a new commit.